server/api/core/core.js

```diff
@@ -348,22 +369,20 @@ export default {
     //
     // Set Default Roles
     //
-
+    const defaultAdminRoles = ["owner", "admin", "guest", "account/profile"];
     // we don't use accounts/addUserPermissions here because we may not yet have permissions
-    Roles.setUserRoles(accountId, _.uniq(defaultAdminRoles), shopId);
```

Comment:
Yeah, I have no idea why that `_.uniq` was there.  Presumably from some other time when setting custom roles was possible.  At this point, those role values are hard coded, so there was no need to run that lodash method on them.  (unrelated, that `_.uniq` was using the `underscore` global from Meteor because `lodash` wasn't being imported previously)